lib/utility/parent-port.ts

```diff
@@ -0,0 +1,29 @@
+import { EventEmitter } from 'events';
+import { MessagePortMain } from '@electron/internal/browser/message-port-main';
+const { createParentPort } = process._linkedBinding('electron_utility_parent_port');
+
+export class ParentPort extends EventEmitter {
+  #port: any
```

Comment:
Refine this type please!

Reply:
Refs https://github.com/electron/electron/pull/34980/commits/a82d3223ede85df14b97780ef2bc1cce21e3aa39